api/envoy/config/core/v3/health_check.proto

```diff
@@ -145,6 +145,12 @@ message HealthCheck {
     // <envoy_v3_api_msg_type.matcher.v3.StringMatcher>`. See the :ref:`architecture overview
     // <arch_overview_health_checking_identity>` for more information.
     type.matcher.v3.StringMatcher service_name_matcher = 11;
+
+    // HTTP Method that will be used for health checking, default is "GET"
+    // GET, HEAD, POST, PUT, DELETE, OPTIONS, TRACE, PATCH methods are supported, but making request body is not supported.
+    // CONNECT method is disallowed because it is not appropriate for health check request.
+    // If a non-200 response is expected by the method, it needs to be set in :ref:`expected_statuses <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.expected_statuses>`
```

Comment:
nit: add '.' at the end of the line (see rendered [docs here](https://storage.googleapis.com/envoy-pr/01ed4e3/docs/api-v3/config/core/v3/health_check.proto.html?highlight=healthcheck#config-core-v3-healthcheck-httphealthcheck)).

Reply:
Thanks! I added it!